src/Features/Core/Portable/MakeClassAbstract/AbstractMakeTypeAbstractCodeFixProvider.cs

```diff
@@ -38,10 +38,10 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
         {
             for (var i = 0; i < diagnostics.Length; i++)
             {
-                if (IsValidRefactoringContext(diagnostics[i].Location?.FindNode(cancellationToken), out var classDeclaration))
+                if (IsValidRefactoringContext(diagnostics[i].Location?.FindNode(cancellationToken), out var typeDeclaration))
                 {
-                    editor.ReplaceNode(classDeclaration,
-                        (currentClassDeclaration, generator) => generator.WithModifiers(currentClassDeclaration, DeclarationModifiers.Abstract));
+                    editor.ReplaceNode(typeDeclaration,
+                        (currentTypeDeclaration, generator) => generator.WithModifiers(currentTypeDeclaration, DeclarationModifiers.Abstract));
```

Comment:
This is not working due to lack of support for records in syntax generator.
Being fixed in #48096